recipes/arraylake-client/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "arraylake-client" %}
+{% set version = "0.4.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arraylake_client-{{ version }}.tar.gz
+  sha256: e01b97f779089fde2ec3aae58e32034be8c81a93caec5e4c62a5ac0b778a68fa
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+
+requirements:
+  host:
+    - python >=3.8,<3.11
+    - poetry-core >=1.0.0
+    - poetry-dynamic-versioning
+    - pip
+  run:
+    - python >=3.8,<3.11
+    - aiobotocore >=2.4.0,<3.0.0
+    - anyio >=3.6.1,<4.0.0
+    - zarr >=2.14,<2.15
+    - numpy >=1.20.0,<2.0.0
+    - donfig >=0.7,<1.0
+    - pydantic >=1.10.0,<2.0.0
+    - httpx >=0.23.0,<1.0
+    - pymongo >=4.2.0,<5.0.0
+    - cachetools >=5.3.0,<6.0.0
+    - structlog >=23.1.0,<24.0.0
+    - email_validator
+    - uvloop  # [not win]
```

If this is optional you can make it noarch, and available on Windows ,by adding both OS-noarch selectors. If this is not optional you can still make it noarch but you cannot add the Windows option. See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-packages-with-os-specific-dependencies for more info.

```suggestion
    - uvloop  # [not win]
```